src/OrchardCore/OrchardCore.Workflows.Abstractions/Models/ActivityMetadata.cs

```diff
@@ -0,0 +1,13 @@
+namespace OrchardCore.Workflows.Models
+{
+    /// <summary>
+    /// Provides additional information about an activity.
+    /// </summary>
+    public class ActivityMetadata
+    {
+        /// <summary>
+        /// A custom title that can be set by the user.
+        /// </summary>
+        public string Title { get; set; }
```

Comment:
That's good because in a future release we'll introduce a more complex structure that will handle localized values. So we'll be able to show a localizable text editor.

Reply:
And we’d have to change it one place only thanks to the SectionDisplayDriver.